### ISSUE TITLE
Calculate COSMO activity coffecient for riluzole drug and carbon dioxid

### ISSUE THREAD

**Issue by mohammadamin73** [lvpp/sigma#30]
I  want to do the calculations for the riluzole drug and carbon dioxide. 
 I drew the structure through wxMacMolplt software and then converted it to (file.inp) with optimization and DFT calculation. Then, use from the OPENBABEL software, the file was converted to (file.mol) .The structure of two materials and (file.mol) is attached.

**Comment by rpseng** [comment]
Please reopen the issue and attach the .mol file here if you want this to be considered. Carbon dioxide is already on the database, so no need.

**Comment by mohammadamin73** [comment]
this site does not accept this format when I attach the mole file. The structure of the molfile is in the zip file.
 when I search the name of carbon dioxide in the jcosmo program, The program gives an error "Missing sigma profile data"

**Comment by rpseng** [comment]
OK about the zip. Regarding carbon dioxide it is found as CARBON_DIOXIDE.

**Comment by mohammadamin73** [comment]
thank you. Did you get the .molefile in zip file?

**Comment by mohammadamin73** [comment]
[reregardingaboutcosmoactivitycoefficient.2.zip](https://github.com/lvpp/sigma/files/6388470/reregardingaboutcosmoactivitycoefficient.2.zip)

**Comment by rpseng** [comment]
Yes, we got the file. It will be processed, this can take several days.

**Comment by mohammadamin73** [comment]
Hello Prof.Rafael de Pelegrini Soares
Were calculations performed for Riluzole?

**Comment by mohammadamin73** [comment]
Hello Prof.Rafael de Pelegrini Soares
Were calculations performed for Riluzole?

**Comment by mohammadamin73** [comment]
Hello
So why is there no news of calculations yet? Why is it so long?
